.github/workflows/lint.yml

```diff
@@ -2,32 +2,41 @@ name: lint
 
 on: push
 
+env:
+  POETRY_VERSION: "1.3.1"
```

Comment:
Do we want to use a newer version of `poetry`? It's now on v1.8.2 I think

Reply:
Agreed let's bump the version of poetry.